addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -400,7 +400,7 @@ $o-color-combinations-preset-number: 1;
 // work for one level of nesting. Note: snippets which can contain other
 // snippets will have problem because of this; this is a limitation of the
 // system until a better solution is found.
-$o-color-extras-nesting-selector: '&, .o_colored_level &';
+$o-color-extras-nesting-selector: '&, .o_colored_level &, &.card';
```

Comment:
This it **was** needed to address the scenario:
no preset -> white bg (bs default) -> white text (inherited from the main snippet container with a dak bg) 

Since ~cards~ `s_cards` "require" a preset (otherwise will be transparent), it not necessary indeed ;)
Thanks!